Makefile

```diff
@@ -1,7 +1,7 @@
 # Set make variables from .env file
 ifneq (,$(wildcard ./.env))
-    include .env
-    export
+	include .env
```

Comment:
Huh, I'm surprised this was not caught before. Tabs vs. spaces?

Reply:
Possibly. I changed my editor a bit.